pkg/cli/referrers_cmd.go

```diff
@@ -0,0 +1,140 @@
+//go:build search
+// +build search
+
+package cli
+
+import (
+	"os"
+	"path"
+
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+
+	zotErrors "zotregistry.io/zot/errors"
+)
+
+//nolint:dupl
+func NewReferrersCommand(searchService SearchService) *cobra.Command {
+	searchImageParams := make(map[string]*string)
+
+	var servURL, user, outputFormat string
+
+	var isSpinner, verifyTLS, verbose, debug bool
+
+	imageCmd := &cobra.Command{
+		Use:   "referrers [config-name]",
+		Short: "List images that refer the given image under Subeject field",
```

Comment:
```suggestion
		Short: "List images that refer the given image under Subject field",
```
Same for the `Long` attribute